### ISSUE TITLE
Unable to decrypt devices with Android 10

### ISSUE THREAD

**Issue by ElDavoo** [oguzbakir/twrp_device_xiaomi_olive#1]
Using latest xiaomi.eu ROM. Device is protected by fingerprint backed up by PIN. TWRP used to ask for PIN and decrypt succesfully on android 9.
[log.txt](https://github.com/oguzbakir/twrp_device_xiaomi_olive/files/4874819/log.txt)

**Comment by oguzbakir** [comment]
Yes, we are aware of that issue. When fix is found, it will be published immediately.

Thanks for the log file.

**Comment by ElDavoo** [comment]
Any news on this?
